src/types/types.ts

```diff
@@ -108,6 +108,77 @@ export type HybridSearch = {
   semanticRatio?: number;
 };
 
+export type Locale =
```

Comment:
Are we sure we want to do this? If we add new locales, we have to update this file, right?
cc: @mdubus

Reply:
I'm thinking this should just link to the docs. `ErrorStatusCode` already has this problem. It doesn't contain [`api_key_already_exists`](https://www.meilisearch.com/docs/reference/errors/error_codes#api_key_already_exists) for example, and I'd say it should be removed as well.

Reply:
Ok you convinced me, so let's remove this then. Thanks for your patience @flevi29  🙏

Reply:
Done.